### DISCUSSION TITLE
Capability of implicit solvation method in xtb

### DISCUSSION THREAD

**Discussion by mjhong0708** [grimme-lab/xtb#737] [General]
Hello,

Thanks for developing such a great software.

I'm trying to use xTB in electrolyte system, especially with EC/DEC solvent.
However, it seems that implicit solvation methods implemented in xtb provides limited range of solvents.
Is there a way to specify arbitrary solvent, as in PCM method (ex. with dielectric constant)?
If not, is there any plan to implement other solvation methods, for example, SMD or PCM?
It would greatly help researchers in variety of fields.

**Comment by haneug** [comment]
Hello,
I would recommend picking the solvent most similar to EC/DEC. Additional solvents would need to be fitted for ALPB and GBSA (which requires some work for each solvent added) and we do not plan to add additional solvents in the near future (at least to my knowledge). Regarding COSMO/PCM maybe @MtoLStoN can comment?

**Comment by MtoLStoN** [comment]
I agree with @haneug here. Given the accuracy range of the ALPB method in conjunction with the xTB Hamiltonian, choosing a solvent with a similar DEC is a reasonable assumption.

Regarding your question, there is a ddCOSMO implementation in xTB, which can be invoked with "--cosmo DEC". However, accuracy-wise, I would recommend choosing the ALPB method instead.
